### ISSUE TITLE
Accessiblity Features for Plugin Gallery

### ISSUE THREAD

**Issue by AMStengel8** [pkp/pkp-lib#9961]
## Issue Description
There is not enough information about the plugins in the plugin gallery about the accessibility (or lack thereof) of the plugins. There is also not enough clarity about how recently the plugin was updated, or if it is available in languages other than English. 

## How to reproduce / Current behavior
This lack of information can be viewed in the Plugin Gallery within OJS, by clicking on the plugin's name and viewing the information in the popup box. 

## What application are you using?
 OJS 3.3

## Testing method
- Method: Manual

## Action
// Requesting that information about if a plugin has been tested for accessibility (yes/no) be added to the information in the plugin gallery. An option for plugin developers to include a VPAT or a link to the level of conformance it meets (WCAG 2.1 AA, for example) could also be added. 
Information could also be added about if the plugin is translated into other languges or not, and if so, what languages. 
Information could also be added to clarify if the latest release date for a plugin is the initial release or an update, and if an update, what updates were made. 

## Additional information
As a note, that plugin information is not available to view once the plugin has been installed. This information (accessibility, translation information, and latest release/last updated) would be helpful to view even after the plugin is installed. 
![Plugin Gallery plugin information](https://github.com/pkp/pkp-lib/assets/33467007/0ad76c81-2bae-417e-8807-6103570cd5d2)
